refactor(cdr-select): refactor cdr-select to use Vue 3 <script setup>…

KenjiCrosland opened the pull request:
… syntax

- Add v-model bound to computed selectModel value.
- Add unique id. 
- Remove toArray import from lodash.
- Remove function which computed selected options for a multiple select (v-model takes care of this).
- Fix styling issues with incorrect classes being applied.

wjramos commented:
The only issue I'm seeing so far is column-span elements containing other column-span elements need to be flex as well ( see footer nav for example )

mhewson commented:
Yea I see the need break-point order-changing .

wjramos commented:
I think the scope of this PR should be limited to ensuring all presently available column classes (col-xx-n, col-xx-push-n, col-xx-pull-n, col-xx-offset-n, etc.) 

We also don't need clearfix anymore.

wjramos commented:
@bonpixel @mhewson -- Could you guys try building and see if you see any problems?

mhewson commented:
:+1: ApPrOvE lets get this in and cut a new minor


[![Approved with PullApprove](https://img.shields.io/badge/pullapprove-approved-brightgreen.svg)](https://pullapprove.com/rei/rei-cedar/pull-request/45/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=rei/rei-cedar)

Commit:
refactor(cdr-select): refactor cdr-select to use Vue 3 <script setup> syntax

Add v-model bound to computed selectModel value. Add unique id. Remove toArray import from lodash.
Remove function which computed selected options for a multiple select (v-model takes care of this).
Fix styling issues with incorrect classes being applied.

Commit:
Remove unneeded import
